pytensor/tensor/rewriting/elemwise.py

```diff
@@ -1203,6 +1210,50 @@ def local_careduce_fusion(fgraph, node):
     return [new_car_op(*elm_inputs)]
 
 
+@register_specialize
```

Comment:
Just thinking about where we want this to run:
I guess after any stabilization rewrites, because they might depend on having 1 as an explicit input (eg log(1 + x)).
And probably before the fusion rewrite?

I guess we also want it to run in pre-gradient optimizations?

Reply:
We can only run this after the Fusion rewrite because only Composite Ops allos inlining constants. We cant inline in vanilla Elemwise Ops as they don't have an "inner graph". 

Did you mean something else?

Reply:
Oh I forgot I had registered this in specialize as well. I guess I just wanted to cover developer/used introduced Composites as well? They should be pretty rare, so I can remove it from there.

About gradients... that's a nope. The gradient of Composite is not implemented so we definitely don't want to introduce them before.

Reply:
I removed it from `specialize` to be clear this is a fusion-related rewrite